solution/ui/regulations/eregs-vite/src/components/dropdowns/Categories.vue

```diff
@@ -152,7 +152,7 @@ onMounted(() => {
     window.addEventListener("popstate", onPopState);
 });
 
-onUnmounted(() => window.removeEventListener("resize", onPopState));
+onUnmounted(() => window.removeEventListener("popstate", onPopState));
```

Comment:
this is a bug from the dropdown story -- we were removing the wrong event listener, and not removing the one we needed to remove in order to properly clean up after ourselves